test/sampleCode.test.ts

```diff
@@ -0,0 +1,96 @@
+import * as fs from 'fs';
```

Comment:
I didn't want to go down the rabbit hole of figuring out what the test runner du jour is. This seemed easier. If we extend the test suite, I can use an actual test runner and get rid of all of this test runner cruft.